src/main/java/data_access/MongoLoginUserDataAccessObject.java

```diff
@@ -0,0 +1,48 @@
+package data_access;
+
+import entity.CommonUser;
+import entity.User;
+import use_case.login.LoginUserDataAccessInterface;
+import use_case.signup.SignupUserDataAccessInterface;
+
+public class MongoLoginUserDataAccessObject implements LoginUserDataAccessInterface, SignupUserDataAccessInterface {
```

Comment:
Again, this should probably implement all the interfaces that InMemoryDAO implements